scripts/build_docker_image

```diff
@@ -27,21 +26,4 @@ if bash +x -o nounset -c "$(aws ecr get-login --no-include-email --region "us-we
   else
     exit
   fi
-
-  # build & tag the db migrations image, then push to ECR
-  if (set -x ; docker build --quiet --no-cache --tag "easi-db-migrate" "$builddir" --file Dockerfile.db_migrations) ; then
-    "$scriptdir"/tag_and_push "easi-db-migrate:latest" "${ecr_db_migrate}" "${APPLICATION_VERSION}" || exit
-    "$scriptdir"/tag_and_push "easi-db-migrate:latest" "${ecr_db_migrate}" "latest" || exit
-    "$scriptdir"/tag_and_push "easi-db-migrate:latest" "${ecr_db_migrate}" "dev" || exit
-
-    if [[ "$CIRCLE_BRANCH" = "master" ]]; then
-      "$scriptdir"/tag_and_push "easi-db-migrate:latest" "${ecr_db_migrate}" "impl" || exit
-    fi
-  else
-    exit
-  fi
-else
-  status=$?
-  echo "FATAL: ECR login failed" 1>&2
-  exit "$status"
```

Comment:
Did you mean to delete lines 43-46?

Reply:
Yes, I think these scripts got too complicated and this extra error handling is not useful since aws-cli will print its own error. This isn't a thing that's happening anymore anyway since we got the automation user's access sorted out.

It still needs `else; exit` because, you know, bash. I'll put that back in.

Reply:
Simplifying the script makes sense to me; I had just wanted to make sure the change was intentional.